### ISSUE TITLE
Internal Accessibility Audit and accessibility statement

### ISSUE THREAD

**Issue by calvin-lau-sig7** [alphagov/govuk-brand-guidelines#56]
## What
The brand guideline needs an internal accessibility audit and an accessibility statement. We will take a sample of pages and review against WCAG 2.2.

## Why
It’s a legal requirement

## Who needs to work on this
@selfthinker 

## Done when

- [ ] Identified which pages, sections needs to be audited
- [ ] Do the internal audit
- [ ] Write up issues
- [ ] Draft accessibility statement

**Comment by calvin-lau-sig7** [comment]
Accessibility statement tasks added to new ticket https://github.com/alphagov/govuk-brand-guidelines/issues/94

**Comment by selfthinker** [comment]
I have finished the first part of the audit (the general layout). There are no WCAG fails. But I created GitHub issues as sub-issues for the other types of accessibility issues I found.

I'll move this to 'blocked' now because I need to wait for the 5 relevant pages to be done.

**Comment by manankharwar** [comment]
For your internal accessibility audit process, AccessOps can provide automated WCAG 2.2 scanning to complement manual testing. Might be useful for the systematic review you're planning.
